modules/openy_gc_personal_training/openy_gc_personal_training.install

```diff
@@ -122,3 +122,59 @@ function openy_gc_personal_training_update_8004(&$sandbox) {
     }
   }
 }
+
+/**
+ * Add "State" field to "Personal Training" entity.
+ */
+function openy_gc_personal_training_update_8005($sandbox) {
+  $state_field_storage_definition = BaseFieldDefinition::create('state')
+    ->setLabel(t('State'))
+    ->setDescription(t('The personal training state.'))
+    ->setRequired(TRUE)
+    ->setSetting('max_length', 255)
```

Comment:
Should the state field have `max_length` param? It looks like that it came from textfield example.